tests/program-data/.gitignore

```diff
@@ -0,0 +1,5 @@
+.anchor
```

Comment:
Can remove this file.

Reply:
I removed it but added it back so it can gitignore yarn.lock in the tests repo.